### DISCUSSION TITLE
Fade out of aquariids

### DISCUSSION THREAD

**Discussion by fewarris** [Stellarium/stellarium#1877] [Q&A] [answered]
Hello,

how can I fade out the aquariids?

Thank you in advance!

Kind regards, Felix

**Comment by alex-w** [comment, marked as answer]
Please more details...

**Reply by fewarris** [reply]
I want to make a screenshot of the night sky. I can fade out all the other elements of the universe but not the aquariids. There's still the name tagged to it. How can I fade out the name?

**Reply by alex-w** [reply]
OK, just check settings of Meteor Showers plugin

**Reply by fewarris** [reply]
Til now I haven't installed a Meteor Showers plugin. I only use the 'normal' software. Do I need to install it?
Sorry for that questions I'm a bloody beginner :)

**Reply by alex-w** [reply]
> Til now I haven't installed a Meteor Showers plugin. I only use the 'normal' software. Do I need to install it?
> Sorry for that questions I'm a bloody beginner :)

This plugin is installed and activated by default. Please press Ctrl+Alt+Shift+M and change settings in second tab.

**Reply by fewarris** [reply]
It works!! Thank you very much!! :))
